lib/templates/error.html

```diff
@@ -684,6 +692,9 @@
 qqqqqqqqqqqqqqqqqqoq7cEBCQAAAICg/6/bEagAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
 AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAwEyqo/B+CtGqPQAAAABJRU5ErkJggg==" />
       </div>
+      <div>
+        <img class="tomster-logo" src="https://emberjs.com/images/tomsters/construction-dad5711c.png">
```

Comment:
As you can see above the Ember CLI logo is embedded into the page as a base64 string to e.g. make it work offline. I think we should probably do the same for the Tomster too instead of linking to emberjs.com.

So first we need to download https://emberjs.com/images/tomsters/construction-dad5711c.png, and scale it to 400px height (for retina screens: 2x the 200px that we display). After that we can put it into a base64 converter like https://www.base64-image.de/ to get the string that we need to use instead of the url in the `src` attribute.

Could you do that? I think once that is done this should be good to go.

Did you check how the page looks on a small/mobile screen?

Reply:
I did consider whether or not to do that, but initially, I'd actually wanted to build an array of randomized Tomsters, but... I decided that was not the best approach in the end. Too bloaty. In any case, yes, I think I can do that, and I'll run a multi-browser/mobile check, too.

Reply:
I'm not sure what happened, but the inline version of the tomster seems to have broken feet 🤔 

![bildschirmfoto 2018-02-16 um 10 53 49](https://user-images.githubusercontent.com/141300/36302194-e2cdd926-1307-11e8-9cfa-e589198d2283.png)

Reply:
Thank you for helping me find this! I swear I'm an engineer, but this comment alluded me. I also don't know where Tomster's feet went, but I will find them and restore them to their former glory.

Reply:
It seems to be a base 64 encoding error, though I tried two different sites that returned the same code. I'll give it another go after my day job efforts. :-)